src/components/forms/DownloadAttemptForm/DownloadAttemptForm.tsx

```diff
@@ -0,0 +1,44 @@
+import DefaultInput from '@UI/inputs/DefaultInput/DefaultInput.tsx';
+import styles from '@styles/components/DownloadAttemptForm.module.scss';
+import DefaultButton from '@/UI/buttons/DefaultButton/DefaultButton';
+import { useDownloadAttemptForm } from './hooks/useDownloadAttemptForm';
+
+interface DownloadAttemptFormProps {
+  id: string;
+  onDownload: () => void;
+}
+
+export default function DownloadAttemptForm({
```

Comment:
Мне кажется, что лучше назвать компонент вот так:
```suggestion
export default function UploadAttemptForm({
```

Reply:
Ну, соответсвенно нужно поправить все, что связано с неймингом компонента